js/main.js

```diff
@@ -0,0 +1,48 @@
+const addBooks = document.querySelector('#add');
+const bookName = document.getElementById('title');
+const authorName = document.getElementById('author');
+
+let bookList = [];
+
+function createBook(bookElement) {
+  document.querySelector('#book-list').innerHTML = '';
+  bookElement.forEach((e) => {
+    const element = document.createElement('books');
+
+    element.innerHTML += ` <li>
+        <p>${e.bookNames}</p>
+        <p>${e.authorNames}</p>
+        <input type="button" value="remove" onclick="removeBook(this.id)" id="${bookElement.length}">
+        <hr>
+    </li>`;
```

Comment:
- [ ] Since we are creating `li` tag up there, now remove the `li` tags in this template string.
```suggestion
    element.innerHTML += `
        <p>${e.bookNames}</p>
        <p>${e.authorNames}</p>
        <input type="button" value="remove" onclick="removeBook(this.id)" id="${bookElement.length}">
        <hr>`;
```

Reply:
This issue is well resolved! ✔️